onnxruntime/server/core/metric_registry.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include <prometheus/registry.h>
+
+namespace onnxruntime {
+namespace server {
+struct MetricRegistry {
+  // returns the global metric registry instance
+  // use this handle to get the instance to increment/count requests
+  static MetricRegistry& Get() {
+      static MetricRegistry metricRegistry;
+      return metricRegistry;
+  }
+
+  // Store quantile buckets 
+  static prometheus::Histogram::BucketBoundaries TimeBuckets () {
+    // Anything greater than 10s we should bucket into +inf
+    return {0, 50, 100, 250, 500, 1000, 1000, 2500, 5000, 10000};
```

Comment:
Two 1000?

Reply:
Will fix, this seems to have been an oversight